js/models/LocalSettings.js

```diff
@@ -15,12 +32,22 @@ export default class extends Model {
   }
 
   defaults() {
+    let langDataObj;
```

Comment:
Since this is a string, is it confusing to name it langDataObj?

Reply:
I don't think it's a string. I think it returns the full data object, e.g.:

```
  {
    name: 'Français (French, Canada)',
    code: 'fr_CA',
  },
```

Are you seeing something different?

Reply:
oh ok, i see the confusion. one sec.

Reply:
ok, I needed to use the code from that object rather than the object itself. Good catch that something funky was up.